aws/image/copy-files.sh

```diff
@@ -6,12 +9,18 @@ if [ -e /tmp/files/etc/aa-offline_fs_kbc-resources.json ]; then
 	sudo cp /tmp/files/etc/aa-offline_fs_kbc-resources.json /etc/aa-offline_fs_kbc-resources.json
 fi
 
+if [ -e /tmp/files/etc/certificates/tls.crt ] && [ -e /tmp/files/etc/certificates/tls.key ]; then
+        sudo mkdir -p /etc/certificates
+	sudo cp -a /tmp/files/etc/certificates/tls.crt /etc/certificates/
+	sudo cp -a /tmp/files/etc/certificates/tls.key /etc/certificates/
+fi
+
+# If self-signed certificates, then ca certificate needs to be provided
+if [ -e /tmp/files/etc/certificates/ca.crt ]; then
+	sudo cp -a /tmp/files/etc/certificates/ca.crt /etc/certificates/
+fi
+
```

Comment:
Just confirming since this part is new to me. What happens if there are no certificates at all ? Everything will continue to work but unencrypted ?

Reply:
@bpradipt to the call (i just copied it :P )

Reply:
Yeah, everything will continue to work as-is